app/autotranslate/client/lib/autotranslate.js

```diff
@@ -71,7 +71,7 @@ export const AutoTranslate = {
 		Tracker.autorun(() => {
 			Subscriptions.find().observeChanges({
 				changed: (id, fields) => {
-					if (fields.hasOwnProperty('autoTranslate')) {
+					if (fields.hasOwnProperty('autoTranslate') || fields.hasOwnProperty('autoTranslateLanguage')) {
```

Comment:
@vickyokrm this looks as if the change of language was not observed anymore. Why did you remove this?

Reply:
It is observed.

Reply:
sorry, wrong change mode